common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -100,7 +102,7 @@ public void onNpcDialogue(NpcDialogEvent e) {
             scheduledAutoProgressKeyPress = null;
         }
 
-        if (autoProgress) {
+        if (autoProgress && e.isAutoProgressing()) {
```

Comment:
Shouldn't this test be inverted? If this means "NPC chat event is nonBlocking", then we should not schedule a sneak key press...